packages/docs/src/stories/atoms/Card.stories.tsx

```diff
@@ -46,6 +47,50 @@ Footer.args = {
   )
 }
 
+/**
+ * The `footer` can also be set when card has gap set as `none`.
+ * Example: we want to show a table with a footer, insider a rounded card
+ */
+export const FooterNoGap: StoryFn<typeof Card> = (args) => (
+  <Card {...args}>
+    <Table
+      thead={
+        <Tr>
+          <Th>NAME</Th>
+          <Th>EMAIL</Th>
+          <Th>STATUS</Th>
+        </Tr>
+      }
+      tbody={
+        <>
+          <Tr>
+            <Td>M. Jordan</Td>
+            <Td>mjordan@email.com</Td>
+            <Td>active</Td>
+          </Tr>
+          <Tr>
+            <Td>B. Wayne</Td>
+            <Td>bwayne@email.com</Td>
+            <Td>active</Td>
+          </Tr>
+        </>
+      }
+    />
+  </Card>
+)
+FooterNoGap.args = {
+  overflow: 'visible',
```

Comment:
This needs to be `'hidden'`.

Just thinking that when `noGap` is set, hidden should be the default to avoid inner content being visible near the corners. Does it make sense?

<img width="56" alt="Screenshot 2024-01-11 alle 16 24 01" src="https://github.com/commercelayer/app-elements/assets/1681269/ac78050d-83f2-4195-b969-007ecae70125">

Reply:
Spot on!
I've updated the interface to disallow overflow prop in case of `gap: none`